lib/io/stream/writable-object.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Interface for core functionality of write streams.
+ */
+export interface WritableObject {
+    write(chunk: string);
```

Comment:
Should this interface be ever used with file system, there will be an issue with asynchronous operations. This function should definitely return a promise.

Reply:
I tried to make it fit to Node.js streams where the write function is synchronous.

Reply:
I see the point, but the interface should be used in both NodeJS and Browser. So while it is easy to call synchronous stream write function from async method, it is hard to call asynchronous stream write from synchronous method.